flake.nix

```diff
@@ -1,40 +1,36 @@
 {
-  description = "SAC development environment";
+  description = "SaC Compiler and Stdlib";
 
-  inputs.nixpkgs.url = github:NixOS/nixpkgs/nixpkgs-unstable;
-  inputs.flake-utils.url = github:numtide/flake-utils;
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
 
-  outputs = { flake-utils, self, nixpkgs }:
-    # sac Version Components
-    let sacVCs = {
-          version = "1.3.3";
-          vname = "MijasCosta";
-          changes = "572";
-          rev = "1";
-          commit = "g9eca";
-        };
-    in
-    flake-utils.lib.eachDefaultSystem ( system:
-      let
-        pkgs = nixpkgs.legacyPackages.${system};
-      in
-        with pkgs; rec {
-          packages = flake-utils.lib.flattenTree rec {
-            libxcrypt = callPackage ./libxcrypt { };
-            # TODO: should this be smtng like sac.compiler?
-            sac2c = callPackage ./sac2c { inherit sacStdLib sacVCs libxcrypt; };
-            sacStdLib = callPackage ./stdlib { inherit sacVCs; };
-          };
-          defaultPackage = packages.sac2c;
-          apps.repl =
-            flake-utils.lib.mkApp {
-              drv = pkgs.writeShellScriptBin "repl" ''
```

Comment:
I don't remember why I had this; it looks copied from somewhere, at least structurally, but I didn't leave a comment. So I'm not sure right now if it was there for a good reason or if one can just remove it and whatever its functionality was can be achieved in a more canonical way

Reply:
I didn't understand what this was meant to do either, didn't seem like it was directly related with the build, so I removed it.

Reply:
Ok, found it: https://github.com/NixOS/nix/issues/3803 Apparently this is no longer needed